repositories/PageRepository.test.js

```diff
@@ -357,8 +357,8 @@ describe("PagesRepository", () => {
         buildPage({ sectionId: section.id, position: 1 })
       );
 
-      // by moving 10 times the `order` values will converge - (1000 / 2^10) < 1
-      const plan = times(10, i => (i % 2 === 0 ? page1 : page2));
+      // by moving more than 10 times the `order` values will converge, since (1000 / 2^12) < 1
+      const plan = times(12, i => (i % 2 === 0 ? page1 : page2));
```

Comment:
What was the reason for changing this? Moving 10 times and the values would also converge, right? since 2^10 = 1024 and 1000/1024 would still be < 1?